frontend/public/components/overview/project-overview.tsx

```diff
@@ -232,20 +247,6 @@ const ProjectOverviewListItem = connect<ProjectOverviewListItemPropsFromState, P
       <Status item={item} />
     </div>;
 
-    const onClick = (e: Event) => {
-      // Don't toggle details if clicking on a link inside the row.
-      const target = e.target as HTMLElement;
-      if (target.tagName.toLowerCase() === 'a') {
```

Comment:
Not sure if we can totally remove this logic because there are other links in the row like pod links. If we switch the name link to a `btn-link`, we can probably keep this.